examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -400,23 +399,13 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t, colidx)
         dif_flux_ρq_tot_bc,
         diffuse_momentum,
         coupled,
-        ᶠp,
         z_bottom,
         params,
     ) = p
     (; uₕ_int_local, surface_normal) = p
 
-    ᶠgradᵥ = Operators.GradientC2F() # apply BCs to ᶜdivᵥ, which wraps ᶠgradᵥ
-
     uₕ_int = Spaces.level(Y.c.uₕ[colidx], 1)
     @. uₕ_int_local[colidx] = Geometry.UVVector(uₕ_int)
-    @. ᶠp[colidx] = ᶠinterp(ᶜp[colidx])
-    @. ᶠK_E[colidx] = eddy_diffusivity_coefficient(
-        C_E,
-        norm(uₕ_int),
-        z_bottom[colidx],
-        ᶠp[colidx],
-    )
 
     if !coupled
```

Comment:
I think we don't need the `coupled` argument anymore. Will check with the coupler.

Reply:
Actually I think we do.... would it be possible to make the `get_surface_fluxes!(Y, p, colidx)` call conditional based on whether it's coupled? Otherwise I think it will overwrite the fluxes the fluxes in the cache that were saved by the coupler before the atmmos step... Happy to connect to discuss! :)

Reply:
Changed, thanks!